libs/dh/market-participant/actors/feature-delegation/src/lib/dh-delegation-tab.component.ts

```diff
@@ -37,21 +44,35 @@ import { DhDelegationCreateModalComponent } from './dh-delegation-create-modal.c
   ],
   template: `
     <vater-flex *transloco="let t; read: 'marketParticipant.delegation'">
-      <vater-stack direction="row" justify="center">
-        <watt-empty-state
-          icon="custom-no-results"
-          [title]="t('emptyTitle')"
-          [message]="t('emptyMessage')"
-        >
-          <watt-button
-            *dhPermissionRequired="['delegation:manage']"
-            (click)="create()"
-            variant="secondary"
-          >
-            {{ t('emptyStateCreateBtn') }}
-          </watt-button>
-        </watt-empty-state>
-      </vater-stack>
+      @if (isLoading()) {
+        <vater-stack direction="row" justify="center">
+          <watt-spinner />
+        </vater-stack>
+      } @else {
```

Comment:
Suggestion: Could else if not be used here?

Reply:
Probably can. Nice catch :)